core/src/main/resources/reference.conf

```diff
@@ -553,16 +553,48 @@ datastax-java-driver {
   metrics {
     # The session-level metrics that are enabled. See CoreSessionMetric in the codebase for a
     # description of each.
-    session.enabled = [
-      cql_requests,
-    ]
+    session {
+      enabled = [
+        cql_requests,
+      ]
+
+      # Extra configuration (for the metrics that need it)
+      cql_requests {
+        # The largest latency that we expect to record (should be consistent with request.timeout).
+        #
+        # This is used to scale internal data structures. If a higher recording is encountered at
+        # runtime, it is discarded and a warning is logged.
+        highest_latency = 2 seconds
```

Comment:
Shouldn't we default then to `${request.timeout}`?

Reply:
👍 
I wonder if there is a risk of observing a higher value because of some processing overhead though. With significant_digits = 3 it would have to be a few milliseconds more, so I think we're safe. At worst it will come up as a warning and we can advise users to configure it a bit higher.